Atomic/mount.py

```diff
@@ -86,6 +87,10 @@ def set_args(self, args):
 
     def mount(self):
         try:
+            d = OSTreeMount(self.mountpoint, live=self.live, shared=self.shared)
+            if d.mount(self.image, self.live):
+                return
+
```

Comment:
Is this normally done where a variable assignment is used followed by a return and then used again ?